### DISCUSSION TITLE
Renovate stopped auto merging branches with no config change

### DISCUSSION THREAD

**Discussion by msallent** [renovatebot/renovate#10247] [Request Help] [answered]
A couple of weeks ago, my branches stopped getting auto merged and now I get a PR for each, even when the config states otherwise.

I thought maybe Travis was causing some issues with the checks so I removed it in favour of GitHub Actions, but nothing changed.

Logs (msallent/gatsby-starter-skeleton: #365775054) show this:

```
DEBUG: Checking if we can automerge branch(branch="renovate/react-17.x")
DEBUG: mergeStatus=automerge aborted - PR exists(branch="renovate/react-17.x")
DEBUG: Branch automerge not possible(branch="renovate/react-17.x")
```

My config file:

```
{
  "extends": ["config:base"],
  "schedule": ["every weekend"],
  "timezone": "America/Buenos_Aires",
  "dependencyDashboard": true,
  "labels": ["dependencies"],
  "packageRules": [
    {
      "matchUpdateTypes": ["minor", "patch"],
      "matchCurrentVersion": "!/^0/",
      "automergeType": "branch",
      "automerge": true
    }
  ]
}
```

Is there any reason why this is happening? I can provide extra logs if needed.

**Comment by HonkingGoose** [comment, marked as answer]
I noticed the bot is still making noise by making a PR (and causing a notification).

Quote from https://docs.renovatebot.com/automerge-configuration/#branch-vs-pr-automerging:

> Add the `renovate/**` branch to your testing workflow files, or Renovate will not work properly with the `automergeType=branch` setting.

I've made a PR on your repository that (I hope) will get your automerge working properly again. See https://github.com/msallent/gatsby-starter-skeleton/pull/106

**Reply by msallent** [reply]
Thank you :)

Seems weird though that it suddenly stopped working while using Travis. I've merged your PR, hopefully it won't have any issues next weekend.

**Reply by HonkingGoose** [reply]
@msallent I think automerge is working properly now? I don't see any pull requests for updates anymore (as we planned), but I do see update commits that were pushed to your `main` branch, so it seems to be working as intended now. 😄

**Reply by msallent** [reply]
@HonkingGoose working like a charm! Thanks for your help 😉

**Comment by rarkins** [comment]
You need to capture the logs when the PR is created, not later on after it's already created. This log says nothing except that the PR was already created previously so now branch automerge is disabled:

DEBUG: mergeStatus=automerge aborted - PR exists(branch="renovate/react-17.x")

**Reply by itsdouges** [reply]
Hello I'm potentially having the same issue, pull requests aren't being auto merged (however they are auto approved). Any ideas?

**Compiled repo**
Not working at all: https://github.com/atlassian-labs/compiled/pull/782

**Website rpo**
Working: https://github.com/compiled/website/pull/235
Not working: https://github.com/compiled/website/pull/228

🤔

**Reply by viceice** [reply]
Pleaseas check the logs

**Reply by itsdouges** [reply]
https://app.renovatebot.com/dashboard#github/atlassian-labs/compiled/411335499 with branch `renovate/pretty-quick-3.x` https://github.com/atlassian-labs/compiled/pull/785

```
DEBUG: Skipping assignees and reviewers as automerge=true(branch="renovate/pretty-quick-3.x")
DEBUG: Created Pull Request #785(branch="renovate/pretty-quick-3.x")
DEBUG: PR is configured for automerge(branch="renovate/pretty-quick-3.x")
DEBUG: PR is not ready for merge(branch="renovate/pretty-quick-3.x")
```

🤔 not sure what to make of it. i'll see if the renovate website has troubleshooting for auto merge.

**Reply by viceice** [reply]
Maybe you've changed some branch protections recently? Maybe a missing required job?

**Reply by itsdouges** [reply]
You might be right! Thank you, I'll investigate.

**Reply by itsdouges** [reply]
Fixed it https://github.com/atlassian-labs/compiled/pull/785#event-5067720290

Came at the result of this:

![image](https://user-images.githubusercontent.com/6801309/126980096-df87ee76-cc01-4cde-9fc9-4de93e13c937.png)

Added renovate and happy days. Cheers again!
